routers/api/v1/repo/release.go

```diff
@@ -68,6 +68,47 @@ func GetRelease(ctx *context.APIContext) {
 	ctx.JSON(http.StatusOK, convert.ToRelease(release))
 }
 
+//  GetLatestRelease gets the latest stable release of a repository
```

Comment:
I think here is the problem the CI has:
```suggestion
// GetLatestRelease gets the latest stable release of a repository
```